mobile/lib/features/welcome/onboarding.dart

```diff
@@ -0,0 +1,249 @@
+import 'package:get_10101/common/scrollable_safe_area.dart';
+import 'package:get_10101/common/snack_bar.dart';
+import 'package:get_10101/features/wallet/wallet_screen.dart';
+import 'package:get_10101/features/welcome/seed_import_screen.dart';
+import 'package:get_10101/features/welcome/welcome_screen.dart';
+import 'package:get_10101/ffi.dart';
+import 'package:get_10101/logger/logger.dart';
+import 'package:get_10101/util/file.dart';
+import 'package:get_10101/util/preferences.dart';
+import 'package:go_router/go_router.dart';
+
+final themeMode = ValueNotifier(2);
+
+class CarouselItem {
+  final String title;
+  final String description;
+  final String imagePath;
+
+  CarouselItem(this.title, this.description, this.imagePath);
+}
+
+final List<CarouselItem> caruselItems = [
+  CarouselItem("Your keys, your control", "Stay in control of your funds at all time.",
+      "assets/carousel_1.png"),
+  CarouselItem("Bitcoin only & Lightning fast.",
+      "The highest level of security, at lightning speed.", "assets/carousel_2.png"),
+  CarouselItem("Perpetual futures trading.",
+      "Experience P2P leveraged trading with no counterparty risk.", "assets/carousel_3.png"),
+  CarouselItem("Hedging and synthetics",
+      "You can now send, receive and hold USDP natively on Lightning.", "assets/carousel_4.png"),
+];
+
+List<Widget> carouselItemWidgetLayers = [
+  carouselItemWidget(caruselItems[0]),
+  carouselItemWidget(caruselItems[1]),
+  carouselItemWidget(caruselItems[2]),
+  carouselItemWidget(caruselItems[3])
+];
+
+Widget carouselItemWidget(CarouselItem item) {
+  return Stack(children: [
+    Padding(
+      padding: const EdgeInsets.fromLTRB(10, 0, 10, 0),
+      child: ClipRRect(
+          borderRadius: const BorderRadius.all(Radius.circular(5.0)),
+          child: Image.asset(
+            item.imagePath,
+            fit: BoxFit.cover,
+            width: 800.0,
+          )),
+    ),
+    Padding(
+      padding: const EdgeInsets.fromLTRB(15, 250, 15, 0),
+      child: Text(
+        item.title,
+        style: const TextStyle(fontSize: 25, fontWeight: FontWeight.bold),
+        textAlign: TextAlign.center,
+      ),
+    ),
+    Padding(
+      padding: const EdgeInsets.fromLTRB(15, 325, 15, 0),
+      child: Text(
+        item.description,
+        style: const TextStyle(fontSize: 20),
+        textAlign: TextAlign.center,
+      ),
+    )
+  ]);
+}
+
+class Onboarding extends StatefulWidget {
+  static const route = "/on-boarding";
+  static const label = "Welcome";
+
+  const Onboarding({Key? key}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() {
+    return _Onboarding();
+  }
+}
+
+class _Onboarding extends State<Onboarding> {
+  int _current = 0;
+  final CarouselController _controller = CarouselController();
+  bool buttonsDisabled = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        body: ScrollableSafeArea(
```

Comment:
Are you saying this because of the AppBar bug where the text is white and one can't read it? I think this does not apply here.

Reply:
I am saying it because of this 🙂 

![Screenshot 2023-11-13 at 16 26 51](https://github.com/get10101/10101/assets/382048/e7b33536-244d-4e07-9d67-6b6f5714b82a)

Reply:
Gotya. It's fixed now: 

<img width="332" alt="image" src="https://github.com/get10101/10101/assets/224613/e64b5987-4c52-4b09-aef3-32fc64315e8b">